src/frontend/src/routes/(new-styling)/unsupported/+page.svelte

```diff
@@ -1,66 +1,12 @@
 <script lang="ts">
-  import { SUPPORT_URL, UNSUPPORTED_BROWSER_SUPPORT_URL } from "$lib/config";
-  import Header from "$lib/components/layout/Header.svelte";
-  import { ArrowRightIcon, TriangleAlertIcon } from "@lucide/svelte";
-  import FeaturedIcon from "$lib/components/ui/FeaturedIcon.svelte";
-  import { t } from "$lib/stores/locale.store";
-  import { Trans } from "$lib/components/locale";
-  import Button from "$lib/components/ui/Button.svelte";
-</script>
+  import XBrowser from "./components/views/XBrowser.svelte";
+  import Generic from "./components/views/Generic.svelte";
 
-<div class="flex min-h-[100dvh] flex-col" data-page="new-unsupported-view">
-  <div class="h-[env(safe-area-inset-top)]"></div>
+  const isX = /\bTwitter/i.test(navigator.userAgent);
```

Comment:
How we we know that the X user agent is called "Twitter"? What if they rename to "Xagent" or something tomorrow?

Reply:
A comment could help to explain how a human could validate that this is the correct condition. 

NCR for this PR